src/webapp/WEB-INF/views/index.jsp

```diff
@@ -0,0 +1,9 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>My team</title>
+</head>
+<body>
+<h1>Hello, mates!</h1>
+</body>
+</html>
```

Comment:
checklist: Don’t push jsp file, jsp dependencies, hello controller and InternalResourceViewResolver resolver() bean to PR.